test/unit/spec/common/jobManager-Spec.js

```diff
@@ -1874,18 +1878,15 @@ define([
          * @param {object} ctx  this context
          * @param {object} args with keys
          *          batchId         batch job ID
-         *          allJobIds       IDs of batch job and all children
          *          listenerArray   listeners expected to be active; array of strings, e.g.
          *                          [jcm.MESSAGE_TYPE.STATUS, jcm.MESSAGE_TYPE.INFO]
          */
         function check_initJobs(ctx, args) {
-            const { batchId, allJobIds, listenerArray } = args;
-            const channelStrings = allJobIds.map((id) => {
-                return ctx.jobManagerInstance._encodeChannel(jcm.CHANNEL.JOB, id);
-            });
+            const { batchId, listenerArray } = args;
```

Comment:
changes reflect moving over from having lots of individual job listeners to having a single batch channel listener for each message type.